### DISCUSSION TITLE
Help! Cookies doesn't work

### DISCUSSION THREAD

**Discussion by AdagioB** [quasarframework/quasar#11874] [General - Components / Directives / etc] [answered]
I'm trying to get the Cookies plugin working with Quasar 2.4.2 in an UMD project, but so far I have no luck

This is the one I'm referring to: https://quasar.dev/quasar-plugins/cookies#read-all-cookies

Probably my biggest issue is not understanding the documentation, where it says the following:

```
// outside of a Vue file
import { Cookies } from 'quasar'
const value = Cookies.get('cookie_name')
```

What does it mean 'outside of a Vue file'?

I'm currently trying to set it up in a html file in the script tag. Stupid me reads it as doing something like this:

```
<html>Lots of html code that is not interesting

<script>
import { Cookies } from 'quasar'

const { useQuasar } = Quasar;

const app = Vue.createApp({
 data()
            {
                return {
                    quasar: useQuasar()
                }
            },
methods:
{
readCookie()
{
let value = Cookies.get('cookie_name')`
}
}
})

app.use(Quasar, {
            config: {
            }
        })


        app.config.debug = true
        app.config.devtools = true

        app.mount('#q-app')
</script>
</html>
```

But this gives me the error Cannot use import statement outside a module. Where do I add this import? 
I have been able to find any documentation that explains this. Only thing I found was a "You're doing it wrong", but not any solution. I know what I'm doing is wrong, but being new to this I have no idea what the correct way is. Hoping some kind soul would help me

**Comment by pdanpdan** [comment, marked as answer]
On the same page you have the usage instructions for UMD
And more details on the global object here: https://quasar.dev/start/umd#quasar-global-object

**Reply by AdagioB** [reply]
The only thing it mentions about UMD is that nothing is needed for install, but thanks the solution was hidden on the global object page. I totally missed it when I read that page. I just needed to write Quasar.Cookies.get('cookie_name')
Thanks a lot for the help, that was exactly what I needed
